### PR TITLE
[Merged by Bors] - feat: computable recursor for Quiver.Path

### PR DESCRIPTION
**Pull request by bottine**
… in order to facilitate the port of `CategoryTheory.PathCategory`.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Combinatorics/Quiver/Path.lean [hunk 1]
```diff
@@ -31,6 +31,21 @@ inductive Path {V : Type u} [Quiver.{v} V] (a : V) : V → Sort max (u + 1) v
   | cons : ∀ {b c : V}, Path a b → (b ⟶ c) → Path a c
 #align quiver.path Quiver.Path
 
+/-- A computable version of `Path.rec`. Workaround until Lean has native support for this. -/
```
Comment (eric-wieser) [inline Mathlib/Combinatorics/Quiver/Path.lean:34]: Can you reference the lean4 issue in a comment above?

Comment (eric-wieser) [reply Mathlib/Combinatorics/Quiver/Path.lean:34]: also,
```suggestion
/-- A computable version of `Quiver.Path.rec`. Workaround until Lean has native support for this. -/
```

**Review by eric-wieser — APPROVED** [review]
bors d+

Thanks!

File: Mathlib/Combinatorics/Quiver/Path.lean (hunk 1 above)
Comment (bottine) [reply Mathlib/Combinatorics/Quiver/Path.lean]: Works like I just did?

**Pull request closed without merging**